client/src/app/modules/login/components/login/login.component.ts

```diff
@@ -0,0 +1,38 @@
+import { Component, OnInit } from '@angular/core';
+import { FormControl, Validators, FormBuilder, FormGroup } from '@angular/forms';
+import { LoginService } from '../../../../shared/services/login.service';
+import { Router } from '@angular/router';
+
+@Component({
+  selector: 'app-login',
+  templateUrl: './login.component.html',
+})
+
+export class LoginComponent implements OnInit {
+  email: string;
+  password: string;
+
+  usernameFormControl: FormControl = new FormControl('', [Validators.required, Validators.email]);
+  passwordFormControl: FormControl = new FormControl('', [Validators.required, Validators.email]);
+
+  constructor(private loginService: LoginService,
+    private router: Router) {
+  }
+
+  ngOnInit() {
+  }
+
+  login() {
+    if (this.email && this.password) {
+      const email: string = this.email;
+      const password: string = this.password;
+
+        this.loginService.login(email, password)
+          .subscribe(() => {
```

Comment:
What happend in case of error ?
If you want to do it later, can you a `// TODO ` comment to not forget about it ?